lib/puppeteer_wrapper/puppeteer_utils.ts

```diff
@@ -11,7 +12,7 @@ export async function stringifyLogText(log: ConsoleMessage): Promise<string> {
 
 function stringifyLogArg(arg: JSHandle): Promise<string> {
     return arg.executionContext().evaluate(element => {
-        if (typeof element === 'object' && !(element instanceof RegExp)) {
+        if (typeof element === 'object' && !isRegExp(element)) {
```

Comment:
FYI This change had to be reverted, as lodash is not available inside browser context.

Because this is only used for log parsing, it should not be a problem for the issue this PR fixes